core/src/repair_service.rs

```diff
@@ -757,7 +797,7 @@ impl RepairService {
 }
 
 #[cfg(test)]
-mod test {
+pub(crate) mod test {
```

Comment:
We could keep this private and elevate `post_shred_deferment_timestamp` outside (and make it `#[cfg(test)]`)

Reply:
that's nicer, changed